stat.proto

```diff
@@ -50,6 +50,8 @@ message StatRequest {
     optional int32 end_point_id = 19;
     optional string end_point_name = 20;
     optional StatInfoResponse info_response = 21;
+    optional int32 token_id = 22;
+    optional string token = 23;
```

Comment:
Additional question (but maybe not relevant): `application_id`, `application_name` and `client` may be linked to that also and redundant?